src/main/java/nl/tudelft/dnainator/ui/widgets/ExceptionDialog.java

```diff
@@ -37,6 +41,10 @@ public ExceptionDialog(Throwable throwable, String title) {
 		content.add(textArea, 0, 1);
 
 		alert.getDialogPane().setExpandableContent(content);
+		Stage stage = (Stage) alert.getDialogPane().getScene().getWindow();
```

Comment:
What is this for? If you want to set an icon for the Stage, please do so in the `DNAInator` class.